crates/bevy_ecs/src/system/observer_system.rs

```diff
@@ -55,37 +53,6 @@ where
     }
 }
 
-macro_rules! impl_system_function {
```

Comment:
Really nice to be able to remove this. Makes the possibility of future "custom" systems much less daunting.